SimplenoteShare/Simperium/Uploader.swift

```diff
@@ -61,10 +61,46 @@ extension Uploader: URLSessionTaskDelegate {
     }
 }
 
+class Downloader: NSObject {
```

Comment:
Maaaybe we should either extract this into its own file, OR just rename as `SimperiumREST.swift` ?

Reply:
moved it to it's own file